pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -16,11 +16,9 @@ stdenv.mkDerivation rec {
   buildInputs = [ python3 xlibsWrapper libxcb libXrandr libXext wayland ];
   enableParallelBuilding = true;
 
-  preConfigure = ''
-    substituteInPlace loader/vulkan.pc.in \
-      --replace 'includedir=''${prefix}/include' 'includedir=${vulkan-headers}/include' \
-      --replace 'libdir=''${exec_prefix}/@CMAKE_INSTALL_LIBDIR@' 'libdir=@CMAKE_INSTALL_LIBDIR@'
-  '';
+  # Set CMAKE_INSTALL_INCLUDEDIR to the vulkan-headers directory
+  include = vulkan-headers;
```

Comment:
This feels like a hack, overriding the variables related to output. Would not setting `cmakeFlags = [ "-DCMAKE_INSTALL_INCLUDEDIR=${vulkan-headers}" ];` be enough.

Reply:
You are right, setting `cmakeFlags` is enough to change the install path. I have tried to do this the first time, but I must have checked the `vulkan.pc` in a outdated `result-dev` link or something, because it didn't work for me. Therefore I came up with this hack.

I will update the PR.